kuma/wiki/tests/test_jobs.py

```diff
@@ -13,6 +13,15 @@ def test_document_zone_unicode(doc_hierarchy_with_zones):
         top_doc.get_absolute_url(), top_doc.title)
 
 
+def test_nearest_zone_expiry():
+    """
+    Ensure that the expiry is not constant.
+    """
+    job = DocumentNearestZoneJob()
+    with mock.patch('time.time', return_value=0):
```

Comment:
I thought this had to be ``mock.patch('kuma.wiki.jobs.time')``, but it appears to be correct (all times are between 1 and 10 days). It is also still fast (``py.test --reuse-db --durations=10 kuma/wiki/tests/test_jobs.py``).